README.md

```diff
@@ -1,23 +1,33 @@
 # eWaterCycle II Project Website
 
-The eWaterCycle II project envisions a collaborative environment containing 
-many local and global hydrological models, allowing hydrologists to run 
-experiments on high performance computing infrastructure, with multiple 
-models, without having to be an expert in computer science. This will all 
-contribute to further advance our knowledge of the global hydrological 
+The eWaterCycle II project envisions a collaborative environment containing
+many local and global hydrological models, allowing hydrologists to run
+experiments on high performance computing infrastructure, with multiple
+models, without having to be an expert in computer science. This will all
+contribute to further advance our knowledge of the global hydrological
 cycle.
 
-# Install the website locally (for development)
+# Install and run the website locally (for development)
 
-```
-git clone git@github.com:eWaterCycle/ewatercycle.github.io.git  
-sudo apt-get install ruby-full  
-sudo gem install bundler  
-sudo bundle install  
-```
+_NB: The following instructions are for Linux.
+On Mac, Ruby version 2.6 may be necessary. You can install a specific version
+with [rbenv](https://github.com/rbenv/rbenv)._
 
-# Run the website locally
-
-```
-bundle exec jekyll serve  
-```
+1. Clone the repository, move to its directory
+    ```
+    git clone git@github.com:eWaterCycle/ewatercycle.github.io.git
+    cd ewatercycle.github.io
+    ```
+1. Install ruby.
+    ```
+    sudo apt-get install ruby-full
+    ```
+1. Install the required gems
+    ```
+    sudo gem install bundler
+    sudo bundle install
```

Comment:
I know it was already there, but when you run this, you get a warning: 

```sh
Don't run Bundler as root. Bundler can ask for sudo if it is needed, and installing your bundle as root will break this application for all non-root users on this machine.
```

Maybe something to keep in mind for future work on similar websites.

Also it complains that `ruby sass is deprecated`. No need to address this now though.

Reply:
This was part of the original readme, but I certainly appreciate removing superfluous sudos from installation instructions.
Flagging this for future fixin' together with someone on Linux :)